Builder image needs updates

## Description

The build container has unapplied security updates, causing builds to fail. 

## Steps to Reproduce
1. Checkout the `develop` branch.
2. Run `make build-debs`.

## Expected Behavior

`make build-debs` tests should pass.

## Actual Behavior

`make build-debs` tests fail:

```
    >       assert dist_upgrade_simulate.rc != 0
    E       AssertionError: assert 0 != 0
    E        +  where 0 = CommandResult(command='apt-get -s dist-upgrade -oDir::Etc::Sourcelist=/tmp/sec...ntu3.3] (1:5.14-2ubuntu3.4 Ubuntu:14.04/trusty-security [all])\n', stderr=None).rc
    
    tests/test_security_updates.py:21: AssertionError
    ============== 1 failed, 175 passed, 1 xpassed in 140.18 seconds ===============
```

--- a/molecule/builder/image_hash
+++ b/molecule/builder/image_hash
@@ -1,2 +1,2 @@
-# sha256 digest quay.io/freedomofpress/sd-docker-builder:2018_07_25
-86cb12fa109798f01c86286cb168a21d5dfe3d676a8dba0c08127338b4dcf07a
+# sha256 digest quay.io/freedomofpress/sd-docker-builder:2018_08_23
+db17d3a734f6a714e57c0b9ce81dec36e51f2ec30a250902b839fb9050cb9d4f

